creator/users/schema.py

```diff
@@ -189,6 +197,7 @@ class Meta:
             "last_name": ["exact"],
             "last_login": ["gt", "lt"],
             "date_joined": ["gt", "lt"],
+            "organizations__name": ["exact"],
```

Comment:
As you can see, graphene converts these to strangely formatted parameters like `organizations_Name`. I'd like to avoid these by explicitly defining the fields on the filter where you can control the name.